test/dotnet-new.Tests/NewCommandTests.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using System;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.New.Tests
+{
+    public class NewCommandTests
+    {
+		private readonly ITestOutputHelper output;
```

Comment:
(double nit) spacing and private fields should start with `_`.  Then you can get rid of `this.` in the constructor too.  :)